static/app/views/performance/landing/widgets/widgetDefinitions.tsx

```diff
@@ -211,14 +212,14 @@ export const WIDGET_DEFINITIONS: ({
   [PerformanceWidgetSetting.SLOW_FRAMES_AREA]: {
     title: t('Slow Frames'),
     titleTooltip: getTermHelp(organization, PERFORMANCE_TERM.SLOW_FRAMES),
-    fields: ['p75(measurements.frames_slow_rate)'],
+    fields: ['measurements.frames_slow', 'p75(measurements.frames_slow_rate)'],
     dataType: GenericPerformanceWidgetDataType.area,
     chartColor: WIDGET_PALETTE[0],
   },
   [PerformanceWidgetSetting.FROZEN_FRAMES_AREA]: {
     title: t('Frozen Frames'),
     titleTooltip: getTermHelp(organization, PERFORMANCE_TERM.FROZEN_FRAMES),
-    fields: ['p75(measurements.frames_frozen_rate)'],
+    fields: ['measurements.frames_frozen', 'p75(measurements.frames_frozen_rate)'],
```

Comment:
A `SingleFieldAreaWidget` only accepts a single field, so why is there 2? And why is one of them a numeric column while the other is an aggregate?

Reply:
They shouldn't be, I thought I removed this before I put it up 👍